src/System.Private.Xml/src/System/Xml/Serialization/Compilation.cs

```diff
@@ -173,6 +173,11 @@ internal static Assembly LoadGeneratedAssembly(Type type, string defaultNamespac
                 name.CodeBase = null;
                 name.CultureInfo = CultureInfo.InvariantCulture;
                 string serializerPath = Path.Combine(Path.GetDirectoryName(type.Assembly.Location), serializerName + ".dll");
+                if (!File.Exists(serializerPath))
```

Comment:
Isn't this touching code in the Xml library? Don't we need to worry about the impact of such changes on the shipping code outside of this tool?